core/src/main/java/org/bitcoinj/params/MainNetParams.java

```diff
@@ -52,6 +52,9 @@ public MainNetParams() {
         packetMagic = CoinDefinition.PacketMagic;
         bip32HeaderP2PKHpub = 0x0488b21e; // The 4 byte header that serializes in base58 to "xpub".
         bip32HeaderP2PKHpriv = 0x0488ade4; // The 4 byte header that serializes in base58 to "xprv"
+        dip14HeaderP2PKHpub = 0x0eecefc6; // The 4 byte header that serializes in base58 to "dpmp".
```

Comment:
from DIP14 (mainnet: 0X0EECEFC5 public, 0x0EECF02E private; testnet: 0x0EED270B public, 0x0EED2774 private)

The pub above doesn't match DIP14.